cms/templates/container_chromeless.html

```diff
@@ -0,0 +1,279 @@
+                                %>
+                                <section class="wrapper-xblock level-page is-hidden studio-xblock-wrapper" data-locator="${xblock_locator}" data-course-key="${xblock_locator.course_key}" data-course-assets="${assets_url}">
+                                </section>
+                                <div class="ui-loading">
+                                    <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">${_("Loading")}</span></p>
+                                </div>
+                            </article>
+                        </section>
+                    </div>
+                </div>
+            </div>
+        </main>
+
+        ##   <div id="page-notification"></div>
+        </div>
+
+        <div id="page-prompt"></div>
+
+        % if context_course:
+        <%static:webpack entry="js/factories/context_course"/>
+        <script type="text/javascript">
+            window.course = new ContextCourse({
+            id: "${context_course.id | n, js_escaped_string}",
+            name: "${context_course.display_name_with_default | n, js_escaped_string}",
+            url_name: "${context_course.location.block_id | n, js_escaped_string}",
+            org: "${context_course.location.org | n, js_escaped_string}",
+            num: "${context_course.location.course | n, js_escaped_string}",
+            display_course_number: "${context_course.display_coursenumber | n, js_escaped_string}",
+            revision: "${context_course.location.branch | n, js_escaped_string}",
+            self_paced: ${ context_course.self_paced | n, dump_js_escaped_json },
+            is_custom_relative_dates_active: ${CUSTOM_RELATIVE_DATES.is_enabled(context_course.id) | n, dump_js_escaped_json},
+            start: ${context_course.start | n, dump_js_escaped_json},
+            discussions_settings: ${context_course.discussions_settings | n, dump_js_escaped_json}
+            });
+        </script>
+        % endif
+        <script type="text/javascript">
+        require(['js/factories/base'], function () {});
+        </script>
+        <%static:webpack entry="js/factories/container">
+            ContainerFactory(
+                ${component_templates | n, dump_js_escaped_json},
+                ${xblock_info | n, dump_js_escaped_json},
+                "${action | n, js_escaped_string}",
+                {
+                    isUnitPage: ${is_unit_page | n, dump_js_escaped_json},
+                    canEdit: true,
+                    outlineURL: "${outline_url | n, js_escaped_string}",
+                    clipboardData: ${user_clipboard | n, dump_js_escaped_json},
+                    isIframeEmbed: true,
+                }
+            );
+        </%static:webpack>
+    </body>
+    <script type="text/javascript">
```

Comment:
[request] There's a lot of boilerplate in this template, along with stuff we probably don't strictly need (like the hotjar piece). That's okay–this isn't intended to be long-lived, and it's a fast way to get up and running. But please use comments to highlight the parts of this template that are specific to the new UI you're building (and not just copied from the older template).

Reply:
Added comment for specific script block and removed the hotjar piece.